.github/workflows/ci-images-garbage-collect.yaml

```diff
@@ -19,6 +19,6 @@ jobs:
       - uses: docker://quay.io/cilium/scruffy:v0.0.3@sha256:ca997451b739cbf03c204cb2523a671c31c61edc606aa5d20dc3560bc7f25bc7
         with:
           entrypoint: scruffy
-          args: --git-repository=./ --stable-branches=origin/main,origin/v1.9,origin/v1.10,origin/v1.11,origin/v1.12,origin/v1.13
+          args: --git-repository=./ --stable-branches=origin/main,origin/v1.9,origin/v1.10,origin/v1.11,origin/v1.12,origin/v1.13,origin/v1.14
```

Comment:
[Nit] Should we remove v1.9 and v1.10 from here?

Reply:
https://github.com/cilium/cilium/pull/26681